keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+        XXXXXXX,        KC_A,         KC_S,   KC_D,   KC_F,   KC_G,   XXXXXXX,
+        EXT_PLV,        XXXXXXX,      XXXXXXX,KC_C,   KC_V,
+                                                      KC_ENT, KC_LGUI,
+                                                              KC_HOME,
+                                               KC_SPC,KC_BSPC,KC_END,
+        // right hand
+             XXXXXXX,   KC_1,   KC_1,   KC_1,   KC_1,   KC_1,             XXXXXXX,
+             XXXXXXX,   KC_1,   KC_1,   KC_1,   KC_1,   KC_1,             XXXXXXX,
+                        KC_Y,   KC_U,  KC_I,   KC_O,   KC_P,             KC_BSLS,
+             XXXXXXX,   KC_H,   KC_J,  KC_K,   KC_L,   KC_SCLN,          KC_QUOT,
+                                  KC_N,    KC_M,XXXXXXX,XXXXXXX,         XXXXXXX,
+             KC_LALT, CTL_T(KC_ESC),
+             KC_PGUP,
+             KC_PGDN,KC_BSPC, KC_ENT
+    ),
+
+/* SuperDuper : https://gist.github.com/narze/861e2167784842d38771
+ *
+ * ,--------------------------------------------------.           ,--------------------------------------------------.
+ * |        |      |      |      |      |      |      |           |      |      |      |      |      |      |        |
+ * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
+ * |        |      |      |      |      |      |      |           |      |      |      |  T←  |  T→  |      |        |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * |        |   A  | [SuperDuper]| Bksp | Gui  |------|           |------|   ←  |  ↓   |   ↑  |  →   | Del  |        |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * |        |      |      |      |      |      |      |           |      |      |      |      |      |      |        |
+ * `--------+------+------+------+------+-------------'           `-------------+------+------+------+------+--------'
+ *   |      |      |      |      |      |                                       |      |      |      |      |      |
+ *   `----------------------------------'                                       `----------------------------------'
+ *                                        ,-------------.       ,-------------.
+ *                                        |      |      |       |      |        |
+ *                                 ,------|------|------|       |------+--------+------.
+ *                                 |      |      |      |       |      |        |      |
+ *                                 | Shift|      |------|       |------|        |Shift |
+ *                                 |      |      |      |       |      |        |      |
+ *                                 `--------------------'       `----------------------'
+ */
+[_SUPERDUPER] = KEYMAP(
```

Comment:
We're phasing out `KEYMAP` in favor of `LAYOUT`.  If you could change this? 


```suggestion
[_SUPERDUPER] = LAYOUT_ergodox(
```